readme.md

```diff
@@ -7,67 +7,257 @@
 
 # Point One Navigation Polaris Service #
 
-This repo provides a library for interacting with the Point One Navigation Polaris Service to receive GNSS RTK corrections. Compatible GNSS receivers can acheive better than 10cm accuracy when used with the service. Supported receivers are currently available from Septentrio and u-Blox. We also offer a hardware SDK you can request by emailing [info@pointonenav.com](mailto:info@pointonenav.com)
+This repo provides C and C++ libraries for interacting with the Point One Navigation Polaris Service to receive GNSS RTK
+corrections. Compatible GNSS receivers can achieve better than 10 cm accuracy when used with the service.
+
+Polaris corrections are distributed using the
+[RTCM 10403 standard](https://rtcm.myshopify.com/collections/differential-global-navigation-satellite-dgnss-standards/products/rtcm-10403-2-differential-gnss-global-navigation-satellite-systems-services-version-3-february-1-2013).
+The may be used with a [Point One Atlas](https://pointonenav.com/devkits) device, or with any RTCM-compatible GNSS
+receiver.
```

Comment:
I did like explicitly mentioning it can be used with a Septentrio or U-Blox receiver.

Reply:
Yeah, I'll add that back. I didn't like the phrasing "supported receivers" - we support any RTCM-capable receiver. We actively test with Septentrio, Novatel, and u-Blox.  Changed as follows:

```
They may be used with a [Point One Atlas](https://pointonenav.com/devkits) device, or with any RTCM-compatible GNSS
receiver (currently tested with [Septentrio](https://www.septentrio.com/en), [Novatel](https://novatel.com/), and
[u-Blox](https://www.u-blox.com/en) receivers).
```